benchmarks/build.xml

```diff
@@ -28,9 +28,11 @@
 			<istrue value="${sample.sql.output.merge}" />
 			<then>
 				<echo>Use ${basedir}/sample-${sample.sql.db.type}.sql to load the database.</echo>
+				<echo>Attention: please make sure SampleSQLBuilder is using the same timezone with portal and database by setting consistent timezone in this property: sample.sql.db.time.zone!</echo>
```

Comment:
@slnn Please output the message to tell the user what the current timezone (value of `sample.sql.db.time.zone`) is, so they can know what they should set for portal and database. In order to print the actual timezone used by SampleSQLBuilder, you may read the property value from the `benchmarks-actual.properties`.

Reply:
OK, got it

Reply:
Hi @dantewang 

If we specify the timezone for SampleSQLBuilder by setting `sample.sql.db.time.zone`, we can get the timezone in the ant by reading the property value from the benchmarks-actual.properties, however, the default value of the property is empty, https://github.com/liferay/liferay-portal/blob/master/benchmarks/benchmarks.properties#L24, if we do not specify the timezone for SampleSQLBuilder then SampleSQLBuilder will use the default timezone of ` java.util.TimeZone`


```
String timeZoneId = properties.getProperty(
					"sample.sql.db.time.zone");

				if (Validator.isNull(timeZoneId)) {
					TimeZone timeZone = TimeZone.getDefault();

					properties.setProperty(
						"sample.sql.db.time.zone, timeZone.getID());
```

So my question is if a user did not specify the value for the property `sample.sql.db.time.zone`, we should just output the empty value of `sample.sql.db.time.zone` or we need output the real timezone that SampleSQLBuilder is using?

This pull just tells users should make sure SampelSQLBuilder uses the same timezone with portal and DB, no matter what timezone SampleSQLBuilder is using.

Lily

Reply:
@slnn benchmarks-actual.properties is not written by SampleSQLBuilder any more?

Reply:
> @slnn benchmarks-actual.properties is not written by SampleSQLBuilder any more?

Yes, the property file came from ant instead of SampleSQLBuilder,  ant passed it to SampleSQLBuilder by `sysproperty` of Ant's java task

`<echoproperties destfile="benchmarks-actual.properties" prefix="sample.sql." />
`

Reply:
In the past, SampleSQLBuilder would write these actual properties into benchmarks-actual.properties, but we removed  this support from the master

Reply:
Please see the changes of https://liferay.atlassian.net/browse/LPS-125421

Reply:
OK, then, just print the attention message.